include/Zweites_Konzept.tex

```diff
@@ -139,17 +139,17 @@ \subsection{Morphologischer Kasten und Variantenbeschreibung}
 
 \subsubsection{Variante 2: Weitflächige Erkennung und Aussortierung (Grün)}
-Es soll durch mehrere Antennen eine grössere Abdeckung erreicht werden und so sichergestellt werden, dass alle Tags ausgelesen werden können. Auch bei dieser Variante soll der Behälter über die Mehrheit der Tags ausgelesen werden und dieser, wenn nötig, aussortiert werden.
+Es soll durch mehrere Antennen eine grössere Abdeckung erreicht und so sichergestellt werden, dass alle Tags ausgelesen werden können. Auch bei dieser Variante soll der Behälter über die Mehrheit der Tags ausgelesen werden und dieser, wenn nötig, aussortiert werden.
 
 \subsubsection{Variante 3: Erkennung und Signalisierung (Blau)}
-Bei dieser Variante soll der Leser bei der Warteposition vor der Einlagerung positioniert werden, da dort die Behälter ein paar Sekunden verbleiben. Danach sollen die erkannten Tags und der Behälter als Mitteilung an die Arbeitsstation geschickt werden (Mail, Middleware, Signallampe). Das mögliche Problem dieser Variante ist, dass zwei Behälter nebeneinander in der Warteposition sein können und dadurch allenfalls Falschpositive generiert werden können.
+Bei dieser Variante soll der Leser bei der Warteposition vor der Einlagerung positioniert werden, da dort die Behälter ein paar Sekunden verbleiben. Danach sollen die erkannten Tags und der Behälter als Mitteilung an die Arbeitsstation geschickt werden (Mail, Middleware, Signallampe). Das mögliche Problem dieser Variante ist, dass zwei Behälter nebeneinander in der Warteposition sein können und dadurch allenfalls Falschpositive generiert werden könnten.
 
 \subsection{Wahl der Variante}
 
-Das Projektteam empfiehlt Variante Zwei, da in den Versuchen die Wichtigkeit der Ausrichtung der Tags festgestellt wurden. Daher werden zwei Antennen verwendet und wie in Abbildung \ref{fig:PosAntennen} dargestellt, damit die zwei Varianten der Bücherlagerung (in Abbildung \ref{fig:LagExemplare} dargestellt) gelesen werden können. Die Aussortierung wird empfohlen, damit der Prozess voll autonom geschieht.
+Das Projektteam empfiehlt Variante Zwei, da in den Versuchen die Wichtigkeit der Ausrichtung der Tags festgestellt wurden. Daher werden zwei Antennen verwendet und wie in Abbildung \ref{fig:PosAntennen} dargestellt, damit die zwei Varianten der Bücherlagerung (siehe Abbildung \ref{fig:LagExemplare}) gelesen werden können. Die Aussortierung wird empfohlen, damit der Prozess voll autonom geschieht.
```

Comment:
Daher werden zwei Antennen verwendet und wie in Abbildung \ref{fig:PosAntennen} dargestellt, damit die zwei Varianten der Bücherlagerung (siehe Abbildung \ref{fig:LagExemplare}) gelesen werden können.

Macht noch keinen sinn

Reply:
![grafik](https://user-images.githubusercontent.com/7289947/59028413-d9dc3680-885b-11e9-9b76-ba1f84077125.png)